### PR TITLE
Adding some missing fields

### PR DESCRIPTION
**Pull request by jrbasso**
Refs #103

### CODE REVIEW

**Comment by shurcooL** [comment]
I haven't verified that the fields are valid, but I've looked at the code changes and not seeing any issues. LGTM if the fields are valid.

File: github/pulls.go [hunk 1]
```diff
@@ -42,7 +42,8 @@ type PullRequest struct {
 	StatusesURL  *string    `json:"statuses_url,omitempty"`
 	DiffURL      *string    `json:"diff_url,omitempty"`
 	PatchURL     *string    `json:"patch_url,omitempty"`
-	Assignee     *User      `json:"assignee,omitempty"` // probably only in webhooks
+	Assignee     *User      `json:"assignee,omitempty"`
+	Assignees    []*User    `json:"assignees,omitempty"`
```
Comment (gmlewis) [inline github/pulls.go]: AFAICT, `Assignees` does not belong on a `PullRequest` struct, but does belong on an `Issue` struct and is already there on [line 39 of `issues.go`](https://github.com/google/go-github/blob/master/github/issues.go#L39).

If you still believe it belongs here, can you please point me to the GitHub documentation where it is mentioned?  Thanks!

Comment (jrbasso) [inline github/pulls.go]: @gmlewis According to [GH API](https://developer.github.com/v3/pulls/#labels-assignees-and-milestones), "Every pull request is an issue, but not every issue is a pull request."
That said, I believe all the fields from issues are applicable to pull requests.

Besides that, if you run the API request to this PR (`curl "https://api.github.com/repos/google/go-github/pulls/411"`) you going to see it there:

image

Comment (gmlewis) [inline github/pulls.go]: Ah!  Very good.  Thank you, @jrbasso!

**Comment by gmlewis** [comment]
LGTM.  Merging.

File: github/pulls.go (hunk 1 above)
Comment (jrbasso) [inline github/pulls.go]: @gmlewis You're welcome.

**Comment by gmlewis** [comment]
Included in https://github.com/google/go-github/commit/56add9d4071bb1a06003a1239ebbc1500692b55d

**Pull request closed without merging**